### ISSUE TITLE
Add option to disable the plugin for local runs

### ISSUE THREAD

**Issue by alexviz7** [Qytera-Gmbh/cypress-xray-plugin#88]
When trying to run cypress headless without the PAT token as it was prior the cypress xray plugin the test run fails.

```
An error was thrown in your plugins file while executing the handler for the before:run event.

The error we received was:

Error: Failed to configure Xray uploader: no viable Xray configuration was found or the configuration you provided is not supported.
You can find all configurations currently supported at https://qytera-gmbh.github.io/projects/cypress-xray-plugin/section/configuration/authentication/
    at initXrayClient (C:\Projects\node_modules\cypress-xray-plugin\src\util\config.js:110:15)
    at beforeRunHook (C:\Projects\node_modules\cypress-xray-plugin\src\hooks.js:53:33)
    at Object.handler (C:\Projects\node_modules\cypress-xray-plugin\plugin.js:12:41)
    at invoke (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_plugins.js:183:18)
    at C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\util.js:59:14
    at tryCatcher (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\bluebird\js\release\util.js:16:23)
    at Function.Promise.attempt.Promise.try (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\bluebird\js\release\method.js:39:29)
    at Object.wrapChildPromise (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\util.js:58:23)
    at RunPlugins.execute (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_plugins.js:164:21)
    at EventEmitter.<anonymous> (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_plugins.js:56:12)
    at EventEmitter.emit (node:events:511:28)
    at EventEmitter.emit (node:domain:489:12)
    at process.<anonymous> (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\util.js:33:22)
    at process.emit (node:events:511:28)
    at process.emit (node:domain:489:12)
    at process.emit.sharedData.processEmitHook.installedValue [as emit] (C:\Users\AppData\Local\Cypress\Cache\12.8.1\Cypress\resources\app\node_modules\@cspotcode\source-map-support\source-map-support.js:745:40)
    at emit (node:internal/child_process:944:14)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```
Only solution is to comment out the cypress xray plugin. 
Is there no way to run the cypress headless without the PAT token?

**Comment by csvtuda** [comment]
This is a nice suggestion, thank you. I will add a `plugin.disable` option which is going to skip everything if set to `true`.

**Comment by alexviz7** [comment]
Thank you for your swift solution, however the test fails still. 
I installed the 3.3.2 version and tried to put plugin.enabled=false either in the configuration or the command line and it fails both ways. In the configuration its red underlined so it means it is a not recognized option yet.

**Comment by csvtuda** [comment]
Sorry, I originally wanted to include your issues and fixes in a single version upgrade to not bombard everyone with version bumps. But the other ones might take more time.

I just released version 3.3.3. You should be able to disable the plugin as described in #89. I will include it in the documentation later on today.

**Comment by alexviz7** [comment]
Perfect cypress run works now without the PAT token.
